### PR TITLE
Jwoo/android gradle update

### PR DESCRIPTION
**Pull request by nesalang**
- Fixed Android Daily Build
- Added Microsoft [Internal Feed](https://microsoft.visualstudio.com/OS/AdaptiveCards/_packaging?feed=AdaptiveCards-Android&_a=feed) for Android

### CODE REVIEW

File: source/android/adaptivecards/build.gradle [hunk 1]
```diff
@@ -36,6 +37,30 @@ signing {
...
+task sourceJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier "sources"
+}
+
+publishing {
+    publications {
+        adaptivecards(MavenPublication) {
+            groupId 'io.adaptivecards'
+            version '1.0.1'
```
Comment (nesalang) [inline source/android/adaptivecards/build.gradle]: I'm not sure about the versioning, and left as hardcoded s tring. how should we version it?

Comment (khouzam) [reply source/android/adaptivecards/build.gradle]: Ideally this should take the nuget version from the custom.props and apply it.

---
In reply to: [215369799](https://github.com/Microsoft/AdaptiveCards/pull/1853#discussion_r215369799) [](ancestors = 215369799)

Comment (nesalang) [reply source/android/adaptivecards/build.gradle]: thansks for the good suggestion; will do

---
In reply to: [215424137](https://github.com/Microsoft/AdaptiveCards/pull/1853#discussion_r215424137) [](ancestors = 215424137,215369799)

Comment (nesalang) [reply source/android/adaptivecards/build.gradle]: @gillesk@microsoft.com  changed the version to read from custom.props and concat file date time stamp string to version.

---
In reply to: [215424137](https://github.com/Microsoft/AdaptiveCards/pull/1853#discussion_r215424137) [](ancestors = 215424137,215369799)

**Comment by khouzam** [comment]
>                 }

This should also use the environment variable for the version

---
Refers to: source/android/adaptivecards/build.gradle:77 in a35671d. [](commit_id = a35671d138ec0f3e7dbce4901f5d9f907936e4d9, deletion_comment = False)

**Pull request merged**